student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom)
+    super(age)
+    @classroom = classroom
+  end
```

Comment:
- [ ] [REQUIERED] Good job so far! i think you missed `name` and `parent_permission`, your `def initialize` should like that:

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```